pkg/controller/operators/olm/operatorgroup.go

```diff
@@ -98,6 +100,28 @@ func (a *Operator) syncOperatorGroups(obj interface{}) error {
 	logger.WithField("targetNamespaces", targetNamespaces).Debug("updated target namespaces")
 
 	if namespacesChanged(targetNamespaces, op.Status.Namespaces) {
+		ogLabel := getOperatorGroupLabel(*op)
```

Comment:
Open Question: Should we update the labels after or before updating the operatorgroup?

Reply:
I would say after OG is updated.

Reply:
Addressed in the latest version of the PR.